docker-compose.yml

```
@@ -0,0 +1,41 @@
+  certbot:
+    image: certbot/certbot
+    container_name: certbot
+    volumes:
+      - ./nginx/certbot:/etc/letsencrypt
+      - ./nginx/www:/var/www/certbot
+    networks:
+      - backend
+    entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew; sleep 12h & wait $${!}; done;'"
+
+networks:
+  backend:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)